apiary.apib

```diff
@@ -3335,7 +3338,8 @@ Draft a new invoice.
                         + extended_description: `Some more information about this awesome product` (string, optional, nullable) - Uses Markdown formatting
                         + unit_of_measure_id: `f79d3e04-b8dc-0637-8f18-ca7c8fc63b71` (string, optional, nullable)
                         + unit_price (object, required)
-                            + Include Money
+                            + amount: `123.30` (number, required)
+                            + currency: `EUR` (Currency, optional)
```

Comment:
We should probably make clear which currency it will then rely on I think?

I'm not entirely sure if we really need to make this optional too. Mostly because a price always exists out of an amount & currency.

Reply:
Reasoning for this change can be found on this ticket: https://teamleader.atlassian.net/browse/ATL-2203

Reply:
Not entirely convinced about that reasoning but can't find any counter arguments either.